server/execution/web_api/lobby.py

```diff
@@ -57,5 +59,9 @@ def change_player_status(id: int, tan: str, alerted: bool):
     except KeyError:
         raise NotFound(
             f"Player with TAN '{tan}' does not exist for execution with id {id}")
+    if alerted:
+        Event.player_alerted(execution_id=execution.id,
+                             time=time.current_time_s(),
+                             player=player.tan).log()
     player.alerted = not alerted
```

Comment:
@simohlsen @YIllmann Why do we set `player.alerted = not alerted`?

Reply:
That would be @7erra code. I only relocated it into the component. I dunno

Reply:
For now, I will leave it as is and we should discuss this later.